server/lib/mail/mail-generator.js

```diff
@@ -74,9 +74,18 @@ function transformSubject(subject, bindings) {
 }
 
 function bindPlaceholder(node, bindings, attachments) {
+
   const id = getAttr(node, 'data-id')
   const placeholder = placeholders.find(pl => pl.id === id)
 
+  //Receipt Placeholder only
+  if (id === 'receipt') {
```

Comment:
Could probably just do:
```js
if (id === 'receipt') {
  bindings.receipt = {...bindings}
}
```

Reply:
Yes, later I did think on that.

Reply:
That's ok too, but you can just return `h(placeholder.format(bindings))` i think. `format` should be a function, and if it's not, it doesn't make much sense to return a text node of the bindings object anyway.